File: Practice/programmers_035.py
def solution(N, stages):
    answer = {}
    l = len(stages)
    for i in range(N):
        answer[i+1] = 0 if stages.count(i+1) == 0 else stages.count(i+1)/l
        l -= stages.count(i+1)
    answer = dict(sorted(answer.items(), key=lambda x: x[1], reverse=True))
    return [i for i in answer]

File: JavaScript/programmers_035.js
// set
function solution(numbers) {
    var numArr = []
    for(let i = 0; i < numbers.length - 1; i++) {
        for(let j = i + 1; j < numbers.length; j++) {
            numArr.push(numbers[i] + numbers[j])
        }
    }

    const answer = [...new Set(numArr)]
    return answer.sort((a, b) => a - b);
}